setup.cfg

```diff
@@ -0,0 +1,50 @@
+[metadata]
+name = airbase
+version = 0.3.0
+author = John Paton
+description = An easy downloader for the AirBase air quality data.
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/johnpaton/airbase
+license = MIT
+classifiers =
+    License :: OSI Approved :: MIT License
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
+    Intended Audience :: Science/Research
+    Topic :: Scientific/Engineering
+    Topic :: Scientific/Engineering :: Atmospheric Science
+
+[options]
+python_requires = >=3.7,<4
+install_requires =
+    aiohttp
+    aiofiles
+    tqdm
+packages = find:
+include_package_data = True
+
+[options.packages.find]
+exclude = 
+	tests
+	tests.*
+
+[options.extras_require]
+    docs =
+        sphinx>=4.2.0
+        sphinx_rtd_theme
+        importlib_metadata; python_version < "3.8"
+    test =
```

Comment:
QQ: Why are these entries indented and the rest aren't?

Reply:
I did not noticed the difference on indentation.
I guess that the indentation does not matter here.
I'll check and write again.

Reply:
The indentations can be removed